shared/utils/generate-from-templates.py

```diff
@@ -118,41 +119,46 @@ def _get_csv_dir(self):
         return self.csv_dirs[self.current_oval]
 
     def _get_template_dir(self):
+        # FIXME: _get_template_dir already defined a few lines above!!
```

Comment:
> MG, stupid dynamic languages.

If you use `vim` try the `rope` extension. It will highlight these issues for you.